tuf-spec.md

```diff
@@ -1153,8 +1157,10 @@ as FILENAME.EXT.
   file.  If the new timestamp metadata file has expired, discard it, abort the
   update cycle, and report the potential freeze attack.
 
-**3**. **Download snapshot metadata file**, up to the number of bytes specified
-in the timestamp metadata file.  If consistent snapshots are not used (see
+**3**. **Download snapshot metadata file**, up to the number of bytes
+specified in the timestamp metadata file. If not specified, download up to a
+number of bytes set by the authors of the application using TUF. This may be
```

Comment:
I purposefully say *"a number of bytes"* instead of *"VAR number of bytes"* because we already use *X* and *Y* above, and *Z* below, so I wasn't sure what VAR to use. If others want, we can make the wording consistent. Any thoughts?

Reply:
I noticed that, we could shuffle the variables so that we use _W_, _X_, _Y_ and _Z_.

Reply:
Good idea. Let's wait till we know in what direction this PR should go. If we decide to make `length` mandatory, we don't need the client-defined upper bound.